common/authorization/interceptor.go

```diff
@@ -123,19 +125,31 @@ func (a *interceptor) Interceptor(
 	return handler(ctx, req)
 }
 
+func (a *interceptor) logAuthError(err error) error {
+	a.logger.Error("authorization error", tag.Error(err))
```

Comment:
can we also log the claims information here (it should not contain any sensitive information hopefully) so that its even easier to debug?

Reply:
I like that, but I think authorizer should log such information, not the interceptor.